gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -479,26 +530,45 @@ def main(args):
                 vep_canonical=args.vep_canonical,
                 vep_mane=args.vep_mane,
                 rare_variants_afs=rare_variants_afs,
+            ).write(filtering_groups_res.path, overwrite=overwrite)
+
+        if args.create_per_sample_counts_ht:
+            logger.info(
+                "Calculating per-sample variant statistics for %s...", data_type
             )
-            create_per_sample_counts_ht(mt, filter_groups_ht).write(
+            filter_groups_ht = get_summary_stats_filtering_groups(
+                data_type, autosomes_only=autosomes_only
+            ).ht()
+            vds_load_func = (
+                get_gnomad_v4_vds
+                if data_type == "exomes"
+                else get_gnomad_v4_genomes_vds
+            )
+            mt = vds_load_func(
+                test=test_dataset,
+                split=True,
```

Comment:
Wouldn't here be: `test=test`?

Reply:
I only want to test on the test dataset if `--test-dataset` is passed